src/main/resources/logback.xml

```diff
@@ -11,4 +11,7 @@
     <root level="info">
         <appender-ref ref="STDOUT" />
     </root>
+    <root level="warn">
```

Comment:
Is this necessary? The level defined on line 11 should define the _minimum_ level to log, so you shouldn't need to explicitly add higher log levels.